doc/conf.py

```diff
@@ -418,6 +418,7 @@
     "matplotlib": ("https://matplotlib.org", None),
     "dask": ("https://docs.dask.org/en/latest", None),
     "cftime": ("https://unidata.github.io/cftime", None),
+    "rasterio": ("https://rasterio.readthedocs.io/en/latest", None),
```

Comment:
Should sparse be added? I don't know if there is already a crosslink but one is coming.

https://github.com/pydata/xarray/blob/2cde80bc100e26894f2328098f5b2139e4963e99/xarray/core/common.py#L1455

Reply:
good point. I would do that in that PR, but if you think that is too confusing we can add this here.

Reply:
:+1:  I asked to add it in the other PR, https://github.com/pydata/xarray/pull/4683#issuecomment-744064159